.github/workflows/test.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ ubuntu-18.04, ubuntu-16.04 ]
+        os: [ ubuntu-18.04, ubuntu-20.04 ]
```

Comment:
20.04 was failing before, probably we can get rid if this tests? We have dedicated harvest tests now. WDYT?

Reply:
👍  let's discuss which can be a good approach not to test only on pre-release

Reply:
removing 20.04 for now until we decide how to proceed with this